app/sidekiq/evss/disability_compensation_form/form526_document_upload_failure_email.rb

```diff
@@ -62,17 +55,28 @@ class Form526DocumentUploadFailureEmail < Job
           }
         )
         raise e
+      ensure
+        StatsD.increment("#{STATSD_METRIC_PREFIX}.exhausted")
+
+        cl = caller_locations.first
+        call_location = ZeroSilentFailures::Monitor::CallLocation.new(ZSF_DD_TAG_FUNCTION, cl.path, cl.lineno)
+        zsf_monitor = ZeroSilentFailures::Monitor.new(Form526Submission::ZSF_DD_TAG_SERVICE)
+        user_account_id = begin
+          Form526Submission.find(form526_submission_id).user_account_id
+        rescue
+          nil
+        end
+
+        zsf_monitor.log_silent_failure(log_info, user_account_id, call_location:)
       end
 
       def perform(form526_submission_id, supporting_evidence_attachment_guid)
         super(form526_submission_id)
-        submission = Form526Submission.find(form526_submission_id)
+
+        submission = form526_submission(form526_submission_id)
```

Comment:
Nit: instead of memoizing this you could look it up directly on this line, since it gets passed to the send mailer method [here](https://github.com/department-of-veterans-affairs/vets-api/pull/19097/files#diff-ea6688680224a44a11eb8e99238f96bbe98b8e07a1cb7b2779280e8f523faae8R79), and then pass it to `log_mailer_dispatch` [here](https://github.com/department-of-veterans-affairs/vets-api/pull/19097/files#diff-ea6688680224a44a11eb8e99238f96bbe98b8e07a1cb7b2779280e8f523faae8R105) since I think that's the only place you are re-referencing it as `@form526_submission` [here](https://github.com/department-of-veterans-affairs/vets-api/pull/19097/files#diff-ea6688680224a44a11eb8e99238f96bbe98b8e07a1cb7b2779280e8f523faae8R115). I might be missing something obvious but not sure what the memoization gets us here

Reply:
Addressed in [f6482d4](https://github.com/department-of-veterans-affairs/vets-api/pull/19097/commits/f6482d42ae750007e456da232ee89a3bf55938d3). Thanks.